_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -110,14 +111,14 @@ The `headers` attribute on the cell refers to a `th` element within the same `ta
 
 #### Passed Example 3
 
-The `headers` attribute on the data cells in the second row refers to a `td` element with a role of `columnheader` within the same `table`.
+The `headers` attribute on the data cells in the second row refers to a `th` element with a role of `columnheader` within the same `table`.
 
 ```html
 <table>
 	<thead>
 		<tr>
-			<td role="columnheader" id="header1">Projects</td>
-			<td role="columnheader" id="header2">Objective</td>
+			<th role="columnheader" id="header1">Projects</th>
+			<th role="columnheader" id="header2">Objective</th>
```

Comment:
Probably don't need the role anymore if we do this. If we're going to fail `headers` referencing `td` elements we should have a new failed example for it. What we have now references an element inside the cell, which I do think is a problem.